spec/01-lexical-syntax.md

```diff
@@ -499,10 +499,7 @@ of the escape sequences [here](#escape-sequences) are interpreted.
 > ```
 >
 > Method `stripMargin` is defined in class
-> [scala.collection.immutable.StringLike](http://www.scala-lang.org/api/current/#scala.collection.immutable.StringLike).
-> Because there is a predefined
-> [implicit conversion](06-expressions.html#implicit-conversions) from `String` to
-> `StringLike`, the method is applicable to all strings.
+> [scala.collection.StringOps](https://www.scala-lang.org/api/current/scala/collection/StringOps.html#stripMargin:String).
```

Comment:
OK, but my comment was that this example shouldn't be in the spec and certainly shouldn't link to some random scaladoc which changes all the time. There's no mechanism for even checking if the links are bad.